hackernews.el

```diff
@@ -300,6 +383,7 @@ This is intended as an :annotation-function in
             ((< x 0) -1)
             (t        0))))
   "Compatibility shim for `cl-signum'.")
+
```

Comment:
No need for this whitespace change, as `hackernews.el`, like e.g. `lisp/simple.el` from the Emacs sources, uses `\n\f\n` instead of `\n\n\f\n` to delimit pages. See also [`(emacs) Pages`](https://www.gnu.org/software/emacs/manual/html_node/emacs/Pages.html).